src/magento/magento-marketplace.md

```diff
@@ -11,38 +14,60 @@ All extensions available on Magento Marketplace have passed an extensive review.
 
-## Installation Process
+## Marketplace credentials
 
-The installation process for Marketplace purchases is different for _on premise_ installations of Magento Commerce than for Magento installations hosted on [Adobe Commerce Cloud][4].
+Before you can install an extension purchased from Magento Marketplace, sign in to your Magento account and verify that you have an active access key. You can sign in to your Magento account from the header of [Marketplace][1] or [Magento.com][6].
 
-![]({% link images/images/magento-marketplace.png %}){: .zoom}
-_[Magento Marketplace][1]_
+Your access key is a set of public and private keys that is used to synchronize your Magento installation with your Magento account and verify your credentials. After your account is synchronized, you must enter your private key each time you install an extension or module from Magento Marketplace or upgrade your Magento installation.
+
+You can create multiple access keys for different purposes, and enable or disable them as needed. However, you must use the same access key that was used to install the Magento software. For example, you cannot use a Magento Open Source access key to update or upgrade Magento Commerce, or vice versa. You also cannot use an access key that belongs to another user, or one that is from a [shared account]({% link magento/magento-account-share.md %}).
+
+### Create an access key
+
+1. Sign in to your Magento account.
+
+1. On the _My Account_ page, choose the **Marketplace** tab.
+
+1. In the upper-right corner next to your name, click the down arrow and choose **My Profile**.
 
-### Adobe Commerce Cloud
+    ![]({% link images/images/magento-marketplace-profile.png %}){: .zoom}
+    _Your Marketplace Profile_
 
-For installations of Magento that are hosted on Adobe Commerce Cloud, Marketplace extensions can be added only from a development environment where the integration can be tested and deployed to a staging environment before being pushed to production. To learn more, see [Install, manage, and upgrade extensions][5].
+1. On the _Marketplace_ tab under _My Products_, click **Access Keys**. Then, do one of the following:
 
-### "On Premise" Magento Commerce
+    - Check to see if you already have a set of access keys for your Marketplace purchases. You can create multiple sets of access keys for different purposes.
 
-Follow these steps to install your Marketplace purchases to an installation of Magento Commerce that resides on an _on premise_ server. Although your server might be physically located in your facility or elsewhere, the term _on premise_ refers to any non-cloud installation of Magento Commerce. As a best practice, extensions should be installed and tested in a development environment before being pushed to production.
+    - Click **Create a New Access Key**. Enter a name for the new key pair and click **OK**. Valid characters include upper- and lowercase characters, and hyphens instead of spaces.
```

Comment:
```suggestion
    - Click **Create a New Access Key**. Enter a name for the new key pair and click **OK**. Valid characters include upper- and lowercase characters and hyphens instead of spaces.
```